ibis/expr/api.py

```diff
@@ -1743,6 +1745,25 @@ def get_backend(expr: Expr | None = None) -> BaseBackend:
     BaseBackend
         The Ibis backend.
 
+    Examples
+    --------
+    >>> import ibis
+
+    Get the default backend.
+
+    >>> ibis.get_backend()  # doctest: +ELLIPSIS
+    <ibis.backends.duckdb.Backend object at 0x7fffbd7e1ee0>
```

Comment:
```suggestion
    <ibis.backends.duckdb.Backend object at 0x...>
```

Reply:
I was just taking a look at that, found an example at:

https://github.com/ibis-project/ibis/blob/161c61f597b323e5f2c6b3085b545b9132514e2d/ibis/backends/impala/__init__.py#L159-L160